Setup.pm

```diff
@@ -110,7 +110,7 @@ sub create_internet_shortcuts {
 }
 
 sub create_shortcuts {
-    my $target  = $Config{perlpath}; # TODO: ensure this is correct
+    my $target  = "cmd /c state activate $NAMESPACE";
```

Comment:
I understand the AC a little differently: From what I understand, we should just run `state activate` but from the directory where the installed project is checked out. That should just be the current directory at the time this function is called...

The command as it is, would potentially check out the project again in a new location.  And the user will need to go to a prompt.  Maybe that is also okay, @rawktron ?

Reply:
Would the following be ok?:
```
use Cwd qw(cwd);

# ...

my $curDir = cwd;
my $target  = "cmd /c  cd $curDir && state activate";
```

I'm not 100% on the quoting regarding command sent to cmd, but I think it should work.

Reply:
If that works with the quoting, that should be fine.  But you could also change the `create_shortcut` function to have a new argument for the working directory cf., https://metacpan.org/pod/Win32::Shortcut

Reply:
Nice, I do prefer that.